honeycomb-guide/src/definitions/betaf.md

```diff
@@ -1 +1,24 @@
 # Beta Functions
+
+Each combinatorial map of dimension *N* contains *N* beta functions
```

Comment:
beta or the greek version as below?

By the way, why only *N*? You use only 2 or 3?

Reply:
- when referring to the beta functions as a set, I prefer to spell out the word rather than use the Greek character in a "standalone" way. Alternatively, I could write "the *β<sub>i</sub>* functions", but then I have to give information on _i_.
- The idea is to define it with N and narrow down to our use case. I'll add an example in the paragraph.